### PR TITLE
noscript tag

### PR DESCRIPTION
**Pull request by flammie**
fix for issue #414

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -21,4 +21,5 @@
 <body>
   <div id="react-mount"></div>
   <script type="text/javascript" src="./index.js?{{CACHE_BREAK}}"></script>
+  <noscript>You cannot use apertium without javascript!</noscript>
```
Comment (sushain97) [inline src/index.html:24]:
```suggestion
  <noscript>You cannot use this website without JavaScript!</noscript>
```

**Review by sushain97 — APPROVED** [review]

**Review by sushain97 — APPROVED** [review]

**Comment by unhammer** [comment]
Are there non-js sites we could link to? ( http://aplica.prompsit.com/en/ doesn't seem to work any more. ) or maybe just a link to https://wiki.apertium.org/wiki/Interfaces and non-js users might choose to install packages or simpleton etc.

**Comment by xavivars** [comment]
Does anyone have a sense of the % of people that may be browsing without JS?

Few years ago, I would have pushed not only to have a link, but even to provide a non-JS alternative. But with all the improvements that JS has brought, the JS-framework adoption & SPAs approach that most sites have taken, it's hard to "browse internet" with JS disabled.

**Comment by unhammer** [comment]
Probably not many, but [once in a while it can be essential to someone](https://shkspr.mobi/blog/2021/01/the-unreasonable-effectiveness-of-simple-html/). If anyone actually makes a non-js version, we should at least link to it in the noscript tag :)

**Pull request merged**